docs/tutorials/projects/modernize-retro-robot.md

```diff
@@ -3,10 +3,608 @@ title: "Modernize a 1980s robot"
+</div>
+
+At the top of the torso, you'll notice a large printed circuit board that was the originally the main digital control center of the Omnibot 2000.
+We'll not use it at all as we modernize MAIV, but we can leave it intact.
+
+Now, using wire cutters and quick wire connectors, connect the battery, switch, and charger wires as shown.
+
+<div class="td-max-width-on-larger-screens">
+<img src="../../img/maiv/maiv_power_wiring.jpg" alt="Omnibot MAIV power wiring." class="alignright" title="Omnibot MAIV power wiring." style="max-width: 400px;" />
+</div>
+
+One 3-terminal quick connect joins the negative wire from the battery, the negative wire from the charging port, and a new wire that you need to run from the torso into the base to power the robot.
+
+Another 3-terminal quick connect will join one end of the power switch, the positive wire from the battery, and the positive wire from the charging port.
+
+Finally, run the unconnected wire from the switch (the positive to power the robot) into the base through one of the holes in the top portion of the base (in the picture we have a 2-terminal quick connect in order to extend the length of our wire - yours is likely long enough to avoid this).
+
+#### Connect the power sources to the motors and Pi
+
+Working in the robot base, use a 4-terminal quick connector to connect the positive power supply wire (from the previous step) to the step down converter *positive* input wire.
+Using a length of 18 gauge wire, connect another terminal in this quick connector to the 12V VCC screw terminal on the L298N motor driver to which you previously connected the base motors and pi.
+One of the terminals to the quick connector is empty. This is expected, you will use it later.
+
+Now use another 4-terminal quick connector to connect the negative power supply wire from the torso to the step down converter *negative* wire.
+Using a length of 18 gauge wire, connect another terminal in this quick connector to the GND screw terminal on the L298N motor driver to which you previously connected the base motors and pi.
+As with the positive connector, one of the negative terminals will be left empty.
+
+Finally, plug the USB-C output from the step down converter to the Raspberry Pi.
+
+#### Power MAIV on
+
+Now you can test that your the wiring is correct.
+Ensure the power switch you just wired is off, and your battery is charged.
+Plug the battery into the barrel connector inside the battery compartment, then turn on the power switch.
+You should see LEDs on the motor driver and Pi light up!
+
+{{% alert title="Troubleshooting" color= "info" %}}
+
+If you do not see the LEDs light up, turn the power switch off.
+
+Check your wiring, use a multimeter to ensure that your battery has enough power, and that the positive and negative polarity is as expected.
+You can also try unplugging the battery and powering instead with the wall adaptor.
+
+{{% /alert %}}
+
+### Base configuration
```

Comment:
I recommend moving this to a higher order heading because some people will skip over the wiring tutorial and just want to see the code - we should make that easy:
```suggestion
## Add AI and Viam 
```